EN locale updates

JStMorgan opened the pull request:
A small PR with a few English-language description suggestions. Includes the current Smelter descriptions, which were missing from the Frozen branch.

kingarthur91 commented:
Couple issues
1. Given the game already has everything in °c having the shelters in k is just annoying. And personally not a fan of having a hard number on it period as it just invites arguments on how it's not correct
2. Speaking of arguing about correctness. Pyrite is iron sulfide not oxide being converted to acid gas

JStMorgan commented:
So the Smelter descriptions were only missing from _my_ copy of the Frozen branch! Rebased, removed references to specific temperatures and updated the changelog.
